src/Common/Models/FileTableRow.h

```diff
@@ -0,0 +1,27 @@
+#ifndef __FILE_TABLE_ROW_H_
+#define __FILE_TABLE_ROW_H_
+
+#include <map>
+#include <string>
+
+#include "../Enums/DataTypes.h"
+
+namespace FQL
+{
+    class FileTableRow
```

Comment:
Can we omit the `File` prefix from the general classes and only leave it in the `FileTableLoader`? 😀